### PR TITLE
Add drag icon for cards stack

### PR DESCRIPTION
**Pull request by Artem4590**
* Resolves: #2669
* Target version: master 

### Summary

Add drag icon for cards stack

**After fix :**

dnd-l

dnd-d

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: src/components/board/Stack.vue [hunk 1]
```diff
@@ -24,6 +24,14 @@
 <template>
 	<div class="stack">
 		<div v-click-outside="stopCardCreation" class="stack__header" :class="{'stack__header--add': showAddCard }">
+			<Actions v-if="canEdit">
+				<ActionButton
+					id="drag_indicator"
```
Comment (juliushaertl) [inline src/components/board/Stack.vue:29]: An id should only be used once, here it would be used multiple times (once for each list)

**Comment by juliushaertl** [comment]
Design-wise I'm not fully convinced about this. At least it seems to be way to present for an action that is probably used less often. We could just show it on hover which would make it still more discoverable than with the current drag and drop. @nextcloud/designers Any further ideas?

**Comment by Artem4590** [comment]
@juliushaertl, @nimisha-vijay, if the behavior is expected I suggest we will close the PR and the corresponding issue (https://github.com/nextcloud/deck/issues/2669).

**Comment by juliushaertl** [comment]
Yes, agreed on that. Let's close this but we should probably find a way to improve the drag behavior otherwise. I'll comment on the issue for that.

**Pull request closed without merging**